patches/content-public-browser-permission_type.h.patch

```diff
@@ -6,7 +6,7 @@ index 5082c0011543a08f16fb8e9e630bc5f722c0214c..77c134bcaa430d75b98472c189a97571
    IDLE_DETECTION = 18,
    PERIODIC_BACKGROUND_SYNC = 19,
  
-+  AUTOPLAY = 18,
++  AUTOPLAY = 20,
```

Comment:
Unless we're using the value (doesn't look like it), we can probably just remove the `= 20` part (since it'll automatically get incremented by one)

Reply:
agree that will simplify this patch update and avoid potential problems

Reply:
Fixed.